apps/mdn/mdn-aws/infra/inputs.tf

```diff
@@ -35,10 +35,12 @@ variable cloudfront_attachments {
     enabled           = "1"
     distribution_name = "mdn-attachment-cdn"
     aliases.stage     = ""
-    aliases.prod      = "mdn.mozillademos.org,demos.mdn.mozit.cloud"
+    #FIXME: mdn.mozillademos.org is taken
+    #aliases.prod      = "mdn.mozillademos.org,demos.mdn.mozit.cloud"
+    aliases.prod      = "demos.mdn.mozit.cloud"
     acm_arn.stage     = ""
     acm_arn.prod      = "arn:aws:acm:us-west-2:178589013767:certificate/2f399635-126c-4e83-bf43-5ddbd0525719"
```

Comment:
I don't think this cert is up-to-date, so should it be removed?

Reply:
You would be correct, I copied this from when I first refactored this but if you look at how the argument is really called its using `${module.acm_star_mdn.certificate_arn}` of course this needs to change during go live to use the actual cert but that variable is just a default value that is set if the module arg is not set